spec/gateways/ips_spec.rb

```diff
@@ -0,0 +1,24 @@
+describe Gateways::Ips do
```

Comment:
We've tended to call the namespace `Gateway` rather than `Gateways`. I'm not really worried so long as it's consistent across the application, though.